Running this manually I then get:

```pre
/raid/conda3/conda-bld/moose-libmesh_1638889135744/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeho/bin/../lib/gcc/x86_64-conda-linux-gnu/9.3.0/../../../../x86_64-conda-linux-gnu/bin/ld: ./.libs/libmesh_dbg.so: undefined reference to `TIMPI::wait(std::vector<TIMPI::Request, std::allocator<TIMPI::Request> >&)'
```

Just a heads up!

--- a/conda/libmesh/build.sh
+++ b/conda/libmesh/build.sh
@@ -74,7 +74,7 @@ fi
 
 source $SRC_DIR/configure_libmesh.sh
 export INSTALL_BINARY="${SRC_DIR}/build-aux/install-sh -C"
-LIBMESH_DIR=${PREFIX}/libmesh METHODS="opt oprof devel dbg" \
+LIBMESH_DIR=${PREFIX}/libmesh \
   configure_libmesh --with-vtk-lib=${BUILD_PREFIX}/libmesh-vtk/lib \
                     --with-vtk-include=${BUILD_PREFIX}/libmesh-vtk/include/vtk-${SHORT_VTK_NAME} \
                     $*

--- a/conda/libmesh/meta.yaml
+++ b/conda/libmesh/meta.yaml
@@ -1,5 +1,5 @@
 # Do not use jinja templating (A physical change to this file is required to trigger a build)
-{% set build = 0 %}
+{% set build = 1 %}
 {% set strbuild = "build_" + build|string %}
 {% set version = "2021.11.11" %}
 

--- a/conda/mpich/conda_build_config.yaml
+++ b/conda/mpich/conda_build_config.yaml
@@ -2,7 +2,7 @@
 ## Any changes made will require additional changes to any item above that.
 
 moose_libmesh:
-  - moose-libmesh 2021.11.11 build_0
+  - moose-libmesh 2021.11.11 build_1
 
 SHORT_VTK_NAME:
   - 9.1

--- a/scripts/configure_libmesh.sh
+++ b/scripts/configure_libmesh.sh
@@ -30,11 +30,6 @@ function configure_libmesh()
     exit 1
   fi
 
-  if [ -z "$METHODS" ]; then
-    echo "METHODS must be set in configure_libmesh"
-    exit 1
-  fi
-
   # Preserves capability in update_and_rebuild_libmesh.sh, but this is set in
   # conda/libmesh/build.sh. If not, conda considers it an "unbound variable"
   if [[ -n "$INSTALL_BINARY" ]]; then
@@ -42,6 +37,9 @@ function configure_libmesh()
   else
     export INSTALL_BINARY="${SRC_DIR}/build-aux/install-sh -C"
   fi
+
+  # If METHODS is not set in update_and_rebuild_libmesh.sh, set a default value.
+  export METHODS=${METHODS:="opt oprof devel dbg"}
 
   cd ${SRC_DIR}/build
   ../configure --enable-silent-rules \

--- a/scripts/update_and_rebuild_libmesh.sh
+++ b/scripts/update_and_rebuild_libmesh.sh
@@ -76,7 +76,7 @@ else
 fi
 
 # If the user set METHOD, but not METHODS, we'll let METHOD override
-# METHODS in this script.
+# METHODS in this script. Otherwise, METHODS has a default in configure_libmesh.sh
 if [[ -n "$METHOD" && -z "$METHODS" ]]; then
   export METHODS="$METHOD"
 fi
@@ -86,9 +86,6 @@ VTK_OPTIONS=""
 if [[ -n "$VTKLIB_DIR" && -n "$VTKINCLUDE_DIR" ]]; then
   export VTK_OPTIONS="--with-vtk-lib=$VTKLIB_DIR --with-vtk-include=$VTKINCLUDE_DIR"
 fi
-
-# Finally, if METHODS is still not set, set a default value.
-export METHODS=${METHODS:="opt oprof dbg"}
 
 cd $SCRIPT_DIR/..
 

